PAS-2-Payouts

```diff
@@ -0,0 +1,60 @@
+---
+PAS: 2
+title: Funding & Payouts Process
+status: Draft
+author: Birdo (@itsbirdo)
+created: 23-02-2024
+---
+
+## Summary
+This PAS documentes the process for requesting funds from the bounty for completed work, or work in advance. 
+
+## Abstract
+To run the Paseo testnet, many payments need to be made to individuals / groups / companies for their work running and maintaining the testnet. 
+This PAS covers the processes for payments including frequency, budget tracking, transparency and reporting to the community. 
+
+## Specification
+### Overview
+The bounty curators will be managing a bounty of 125,018 DOT, approx $850k USD at the time of submission. This DOT needs to be allocated to individuals based on their contribution to the network and as per documented in the treasury proposal. 
+
+As per the proposal the breakdown in funds is as follows:
+
+| Amount (DOT) | Amount (USD) | Allocation |
+| ------------ | ------------ | ---------- |
+| 61,773 DOT   | $420,000 USD | is allocated for 3x full time resources to maintain the protocol | 
+| 44,609 DOT   | $303,300 USD | is allocated for nodes and management of them | 
+| 3,309 DOT    | $22,500 USD  | is allocated for open source validator logging |
+| 4,353 DOT    | $29,600 USD  | is allocated for block explorers |
+| 1,588 DOT    | $10,800 USD  | is allocated for curation fees |
+| 9,383 DOT    | $63,800 USD  | is allocated for contingency |
+
+## Payment structure
+Payments will be made to individuals quarterly in advance, over the 12-month funding period. This is tracked in a budget worksheet, documenting all payments made over the year.
+In some cases there could be retroactive payments, if additional services were offered or increased during the quarter - e.g. a validator spun up additional nodes (at request), beyond the forecast amount. 
+
+## Payout process
+Given the nature of the bounty where many individuals or groups will require funding, it makes sense to manage this in a transparent way. 
+2 weeks before the quarter end, an issue will be raised in the Paseo github. Individuals / groups / companies are to comment on the issue, using the following template:
```

Comment:
I wonder if it's not more effective to create individual PR for each payment instead of comments in a single issue. If there is any ongoing discussion for a specific case, that would mess-up  the whole readness of the issue.

One alternative:

Create one directory called `payouts`. The curators can create every quarter a new directory regarding that Quarter. For example `payouts-q2`. Then each beneficiary of the bounty can submit a PR with a new new that contains the infomation listed below. With this each submission can be tracked differently.

(this is a similar process to the validator onboarding)

Wdyt @educlerici-zondax @itsbirdo  ?

Reply:
I do likes Hector approach, I believe it keeps things tidy which makes audits easy.

Reply:
On the other hand, if we end up going for an automated approach, as Paradox has suggested out of this MR, we can still save proof of the transfers in the suggested way.

Reply:
Yep happy with a single PR per individual - helps keep comments and discussion dedicated to that specific payment. Good suggestion.